jarbas_hive_mind/__main__.py

```diff
@@ -0,0 +1,27 @@
+from jarbas_hive_mind import get_listener
+from jarbas_hive_mind.configuration import CONFIGURATION
+
+def main():
+    import argparse
+    parser = argparse.ArgumentParser(description='Start HiveMind as a server')
+    parser.add_argument("--name", help="human readable name")
+    parser.add_argument("--access_key", help="access key")
+    parser.add_argument("--crypto_key", help="payload encryption key")
+    parser.add_argument("--mail", help="human readable mail")
+    parser.add_argument("--port", help="HiveMind port number")
+    args = parser.parse_args()
+    # Check if a user was defined
+    if args.name is not None:
+        from jarbas_hive_mind.database import ClientDatabase
+        with ClientDatabase() as db:
+            db.add_client(args.name, args.mail, args.access_key, crypto_key=args.crypto_key)
```

Comment:
can this be moved into a new script? i don't like credentials being done automatically, but this particular piece of code is also expected to change considerably/expand in functionality, should have a dedicated script that can be easily maintained

Reply:
The script can run without those flags, not defining a new user.
For example, running just ```HiveMind-server``` will start the server in port 5678 and wont add any credentials to the database.

Reply:
right, but this is a security operation and in most setups i would hope the credentials step requires sudo (if not now, sometime in the near future). 

i would like to have those fundamentally separated and force the user to think about what he is doing, adding credentials without thinking just means it will stay in the db forever and become a security risk, so it is my opinion the launcher should not do this step at all.

I would like to have a install script at some point that would handle all this initial setup and prompt/warn the user on the important steps

Reply:
Moved it to the script ```HiveMind-database```